### PR TITLE
Direct KV track 1 customers to track 2 packages

### LINKED ISSUE #9553: Update package descriptions
Please update the release notes of the legacy packages for KeyVault to contain something like this.

```
There are new packages available that replace this one.
Azure.Security.KeyVault.Keys
Azure.Security.KeyVault.Secrets
Azure.Security.KeyVault.Certificates
It is recommended that you move to the new package.
```

### PR DESCRIPTION
**Pull request by heaths**
Resolves #9553 by updating the descriptions customers would see in UI package managers like in Visual Studio. Will also appear in the description on nuget.org, though after the current description (of which an exerpt shows in search results).

### CODE REVIEW

File: sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj [hunk 1]
```diff
@@ -1,7 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>Microsoft Azure Key Vault Core Class Library</Description>
+    <Description>Microsoft Azure Key Vault Core Class Library
+
```
Comment (weshaggard) [inline sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj]: Should consider sharing these common properties in the props file.

File: sdk/keyvault/ci.yml [hunk 2]
```diff
@@ -38,9 +38,13 @@ stages:
     ServiceDirectory: keyvault
     ArtifactName: packages
     Artifacts:
-    - name: Azure.Security.KeyVault.Certificates
-      safeName: AzureSecurityKeyVaultCertificates
-    - name: Azure.Security.KeyVault.Keys
-      safeName: AzureSecurityKeyVaultKeys
-    - name: Azure.Security.KeyVault.Secrets
-      safeName: AzureSecurityKeyVaultSecrets
+    - name: Microsoft.Azure.KeyVault
```
Comment (weshaggard) [inline sdk/keyvault/ci.yml]: Why are you removing the track 2 libraries? Did you intend to add the track 1 libraries or did you want to create another parallel pipeline for the track 1? I would suggest the later which is what we did for keyvault in java (https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/ci.data.yml). 

FYI @mitchdenny

Comment (heaths) [reply sdk/keyvault/ci.yml]: Yes, this change was intentional. We want to only publish track 1 packages with this new message to hopefully direct developers to track 2 packages.

File: sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj (hunk 1 above)
Comment (heaths) [reply sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj]: 1. Why complicate things with a partial property (i.e. the package descriptions are all different sans this blurb)?
2. This is only for track 1 packages.

Comment (weshaggard) [reply sdk/keyvault/Microsoft.Azure.KeyVault.Core/src/Microsoft.Azure.KeyVault.Core.csproj]: OK from a quick glance I thought they were all the same.

File: sdk/keyvault/ci.yml (hunk 2 above)
Comment (weshaggard) [reply sdk/keyvault/ci.yml]: We generally don't want to edit this ci.yml file each time we release a set of track 1 or track 2 libraries we should either list them both and only approve the ones you want to release or create a separate pipeline for track 1. 

With that said if this is the last time we expect to release the track 1 packages and you plan to revert this back then I'm fine with this. Hopefully when you revert this back you can also remove the track 1 code from master.

Comment (heaths) [reply sdk/keyvault/ci.yml]: Yes, I will revert. We don't want the release pipeline to prompt whether to publish these or not each time. Too easy for someone to mistakenly publish them. If, however, DevOps gave guidance - such as conditioning out track 1 packages by default with a build-time variable to opt into publishing them - I could make such changes.

Comment (heaths) [reply sdk/keyvault/ci.yml]: So if you or @AlexGhiondea could sign off, I'll resolve this without merging and push to a feature branch in the upstream repo, kick off a release build, revert the ci.yml change, then submit a PR to master.

**Review by weshaggard — APPROVED** [review]
Changes look good, but as we talked lets not merge to master and instead push them to a feature branch.

**Comment by heaths** [comment]
Done: https://github.com/Azure/azure-sdk-for-net/tree/issue9553

**Comment by weshaggard** [comment]
If you want that branch to get CI and PR validation they you should follow the naming guidelines and call it `feature/issue9553` but otherwise looks good.

**Pull request closed without merging**